audio-analysis/lib/etl/sqlConnection.py

```diff
@@ -44,11 +44,20 @@ def sqlConnectionSetup(engine):
     
     meta.create_all() # Create all tables
 
+#Function checks if url exists in table already
+def urlExistsSpecific(url, engine, column_name):
+    # Check if URL already exists
+    result = engine.execute('SELECT * FROM `labelled` WHERE `url`="' + url + '" AND `pitch` IS NOT NULL')
```

Comment:
Should we pass in `column_name` into the sql query? or is it only used for the print statement?

Reply:
Why do we check if pitch is not null? If it is null we still want to update it right?